proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -114,12 +113,11 @@ message AddWorkflowTaskRequest {
     // (-- api-linter: core::0140::prepositions=disabled
     //     aip.dev/not-precedent: "to" is used to indicate interval. --)
     google.protobuf.Duration schedule_to_start_timeout = 5;
-    string forwarded_source = 6;
-    temporal.server.api.enums.v1.TaskSource source = 7;
```

Comment:
What's the deployment strategy here? Are we expecting forwarding requests to fail during matching node rollout? Can we function without this information?

Reply:
Without this info the forwarded tasks will be treated as original tasks coming from history. This doesn't make any difference when sync-match is possible for the forwarded task. For non-sync-match cases, the task get's spooled in both root and child partitions, but it's fine because history will dedup them and we'll drop extra ones. So no problem is expected, only elevated level of TaskAlreadyStarted errors.